tests/e2e/test_workflow.py

```diff
@@ -0,0 +1,144 @@
+import pytest
+import csv
+import filecmp
+from pathlib import Path
+
+from tests.utils.helpers import hash_digest, execute, print_diffs
+
+ENDPOINT = "https://raw.githubusercontent.com/digital-land/digital-land-python/e2e_test/tests/data/resource_examples/e2e.csv"
+
+
+@pytest.fixture()
+def create_workspace(tmp_path):
```

Comment:
normally fixtures are named in a declarative (?) rather than imperative style, so instead of `create_workspace` which is a command, you could just use `workspace` or something. I read it in my head as "test_workflow is passed a workspace".